tests/integrations/test_mlflow.py

```diff
@@ -159,3 +159,29 @@ def test_mlflow_tensorflow_register(tmp_path, toy_tensorflow_model):
     # all mlflow models will have a 'predict' method
     downloaded_model = Model(registered_model.model_uri)
     assert hasattr(downloaded_model, "predict")
+
+
+@pytest.mark.integration
+def test_mlflow_tensorflow_h5_register(tmp_path, toy_tensorflow_model):
+    # as if model.pkl already existed on disk
+
+    tmp_path.mkdir(exist_ok=True)
+    model_path = tmp_path / "tensorflowtest"
+    toy_tensorflow_model.save(model_path, save_format="h5")
```

Comment:
If the only thing different about this test and the above test is `save_format="h5"`, try using `@pytest.mark.parametrize()` to factor this into a table-driven test: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests